Add yarn, pnpm, and bun for a package manager

ryuapp opened the pull request:
close #135

I havn't executed and verified all commands.
In this docs, commands is not highlighted, `txt` is added instead of `sh`.

yusukebe commented:
Hi @ryuapp 

Awesome work! There are some things I'd like you do:

* Remove `@latest from `pnpm` and `bun`. These don't require it.
* User `bunx` instead of `bun`.

You can refer to the Remix document: https://remix.run/docs/en/main/other-api/create-remix#package-managers

```txt
npm create remix@latest <projectDir>
# or
yarn create remix <projectDir>
# or
pnpm create remix <projectDir>
# or
bunx create-remix <projectDir>
```

And, I relized that we should add Deno's command. It's like following:

```txt
deno run -A npm:create-hono my-app
```

ryuapp commented:
Hello, @yusukebe. Thanks for quick checking.
I removed `@latest` in yarn, pnpm and bun. And I changed `bun` to `bunx` on create command.

> deno run -A npm:create-hono my-app

This is nice idea. I added it.

yusukebe commented:
@ryuapp 

Great! I'll merge it now. Thanks!
